CONTRIBUTING.md

```diff
@@ -0,0 +1,148 @@
+# We would love your help!
+
+Thanks ahead of time for your interest in helping **Open Austin**. In the spirit of open source and social coding, we welcome your involvement in contributing:
+
+- new issues or bugs
+- fixes
+- code expertise to resolve issues
+- and obviously, pull requests  
+
+<br>
```

Comment:
I know it seems hacky. I'm curious what's the correct way to create vertical spacing between sections of grouped content?

Seems like it would look cleaner, with spacing, once rendered.